### PR TITLE
Fix SSR rendering

### PR DESCRIPTION
**Pull request by chipit24**
Related PRs:
https://github.com/ProcessMaker/screen-builder/pull/322
https://github.com/ProcessMaker/vue-form-elements/pull/59

The above PRs need to be merged and the packages re-published and brought into this package to fix SSR.

### CODE REVIEW

File: resources/js/ssr/tests/config.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+          "form": null,
+          "name": "users",
+          "label": "A Record List HERE",
+          "fields": [
+            { "value": "position", "content": "Position" },
+            { "value": "name", "content": "Name" }
+          ],
+          "editable": false
+        },
+        "component": "FormRecordListStatic",
```
Comment (chipit24) [inline resources/js/ssr/tests/config.json]: Other than placing this config into its own file, the only change made was to switch `FormRecordList` to `FormRecordListStatic`.

**Comment by nolanpro** [comment]
@chipit24 That's awesome, Thanks! Rich-text isn't rendering so I added [a rich-text-renderer.vue file](https://github.com/ProcessMaker/connector-send-email/blob/5f8d16641e3ba0ce37c18b5b1a0b6b093345e47c/resources/js/processes/screen-builder/typeEmail.js#L97) that is supposed to work the same way as the static record list but I'm not getting any output. Could you take a look? Here is the config I replaced your config.json to test with. I just copied it from screen builder:
```
[{
    "name": "email rich text",
    "items": [{
        "label": "Rich Text",
        "config": {
            "icon": "fas fa-pencil-ruler",
            "label": null,
            "content": "<h2 style=\"text-align: center;\"><span style=\"color: #27ae60;\"><em><strong>Rich text editor</strong></em></span></h2>",
            "interactive": true
        },
        "component": "FormHtmlEditor",
        "inspector": [{
            "type": "FormTextArea",
            "field": "content",
            "config": {
                "rows": 5,
                "label": "Rich Text Content",
                "value": null,
                "helper": "The HTML text to display"
            }
        }, {
            "type": "ColorSelect",
            "field": "bgcolor",
            "config": {
                "label": "Element Background color",
                "helper": "Set the element's background color",
                "options": [{
                    "value": "alert alert-primary",
                    "content": "primary"
                }, {
                    "value": "alert alert-secondary",
                    "content": "secondary"
                }, {
                    "value": "alert alert-success",
                    "content": "success"
                }, {
                    "value": "alert alert-danger",
                    "content": "danger"
                }, {
                    "value": "alert alert-warning",
                    "content": "warning"
                }, {
                    "value": "alert alert-info",
                    "content": "info"
                }, {
                    "value": "alert alert-light",
                    "content": "light"
                }, {
                    "value": "alert alert-dark",
                    "content": "dark"
                }]
            }
        }],
        "editor-component": "FormHtmlEditor"
    }]
}]
```

**Comment by chipit24** [comment]
@nolanpro It seems `typeEmail.js` has no effect on what's rendered when I run `php test.php`. I removed the file completely and the output was the same; it still uses the default rich text component which renders nothing in SSR.

**Comment by nolanpro** [comment]
@chipit24 strange, it's working in there for FormRecordListStatic. Let me take a look

**Pull request merged**